sw/device/tests/sim_dv/lc_walkthrough_test.c

```diff
@@ -193,7 +193,7 @@ bool test_main(void) {
           "state.");
       dif_lc_ctrl_token_t token;
       for (int i = 0; i < LC_TOKEN_SIZE; i++) {
-        if (kDestState != kDifLcCtrlStateRma) {
+        if (kDestState != (kDifLcCtrlStateRma & 0xf)) {
```

Comment:
Why is this 0xF instead of 0x1F?

Reply:
good catch. be switching to your proposed solution, we no longer need this change anymore.